File: online-judge-solutions/Codeforces/1715A.py
#https://codeforces.com/problemset/problem/1715/A
#Lang		:	Python 3.8
#Time		:	46 ms
#Memory         :	0 KB
for _ in range(int(input())):
    n,m=map(int,input().split())
    print(0 if (n==1 and m==1) else (n-1)+(m-1)+min(n-1,m-1)+1)

File: online-judge-solutions/Codeforces/1715A.cpp
//https://codeforces.com/problemset/problem/1715/A
//Lang		:	GNU C++14
//Time		:	30 ms
//Memory	:	0 KB
#include<iostream>
using namespace std;
int main(){
	int t,n,m;
	cin>>t;
	while(t--){
		cin>>n>>m;
		cout<<(n==1 && m==1?0:(n-1)+(m-1)+min(n-1,m-1)+1)<<endl;
	}
	return 0;
}
